doc/integrator/ogc_api.rst

```diff
@@ -0,0 +1,52 @@
+.. _integrator_ogc_api:
+
+OGC API Features
+----------------
+
+The OGC API is a new standard that defines modular API following REST standard to access spatial data.
+The current implementation is only on the server side and is based on the OGC API - Features standard and only works with the GetFeature request using a bbox filter.
+
+It is currently supported by both MapServer and QGISserver.
+
+The following variables should be added to the docker-compose-lib.yaml file for the 'MapServer' service:
+
+.. prompt:: bash::
+
+    environment:
+        ...
+        - MAPSERVER_BASE_PATH=/mapserv_proxy/mapserver
+        - OGCAPI_HTML_TEMPLATE_DIRECTORY=/usr/local/share/mapserver/ogcapi/templates/html-bootstrap4/
+        ...
+
+
+The following changes has to be done to the Mapfiles to support the OGC API:
+
+.. prompt:: bash::
+
+    MAP
+        ...
+        WEB
+            ...
+            TEMPLATE "dummy""
+            ...
+        END
+        ...
+    END
+
+QGISserver is already configured to support the OGC API and detect automatically if wfs3 path is used.
+
+Landing pages for both MapServer and QGISserver are not supported yet.
+
+OGC API features are accessible through mapserv_proxy, with the following URLs:
+* ``/mapserv_proxy/<ogc-server>/ogcapi/*``: The Mapserver path.
+* ``/mapserv_proxy/<ogc-server>/wfs3/*``: The QGISserver path.
+
+
+OGC API Documentation
+---------------------
+
+Mapserver documentation:
+https://mapserver.org/ogc/ogc_api.html
+
+QGISserver documentation:
+https://docs.qgis.org/3.34/en/docs/server_manual/services/ogcapif.html
```

Comment:
Missing documentation:

- Documentation about MapServer map alias => update the URL in the related OGC Server.
- Remove all special characters in the OGC Server names because they appear in the par of the URL (recommended).

Reply:
Done.